src/helpers/flashingPost.ts

```diff
@@ -3,7 +3,7 @@ export default function (element?: HTMLAnchorElement, position?: number) {
 
   const makePostPulse = () => {
     element.classList.toggle('animate-pulse')
-    setTimeout(() => element.classList.toggle('animate-pulse'), 3000)
+    setTimeout(() => element.classList.toggle('animate-pulse'), 4000)
```

Comment:
3000 clips the animation in the middle, 4000 is enough time to play animation two times